packages/pretty-format/src/collections.ts

```diff
@@ -4,17 +4,16 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {Config, Printer, Refs} from 'types/PrettyFormat';
+import {Config, Printer, Refs} from './types';
 
 const getKeysOfEnumerableProperties = (object: Object) => {
-  const keys = Object.keys(object).sort();
+  const keys: Array<string | symbol> = Object.keys(object).sort();
 
   if (Object.getOwnPropertySymbols) {
     Object.getOwnPropertySymbols(object).forEach(symbol => {
-      //$FlowFixMe because property enumerable is missing in undefined
+      // @ts-ignore: because property enumerable is missing in undefined
       if (Object.getOwnPropertyDescriptor(object, symbol).enumerable) {
```

Comment:
You can use `!` instead of ignoring: `if (Object.getOwnPropertyDescriptor(object, symbol)!.enumerable) {`